esphome/components/sim800l/sim800l.h

```diff
@@ -45,29 +41,36 @@ class Sim800LComponent : public uart::UARTDevice, public PollingComponent {
   void add_on_sms_received_callback(std::function<void(std::string, std::string)> callback) {
     this->callback_.add(std::move(callback));
   }
+  void add_on_connected_callback(std::function<void()> callback) { this->callback_reg.add(std::move(callback)); }
   void send_sms(const std::string &recipient, const std::string &message);
+  void delete_sms();
   void dial(const std::string &recipient);
 
+  void set_rssi_(sensor::Sensor *rssi_);
```

Comment:
Public methods do not end with `_`
A bit more descriptive name. (It is not setting the rssi)

```suggestion
  void set_rssi_sensor(sensor::Sensor *rssi);
```